Xabe.FFmpeg/FFbase.cs

```diff
@@ -71,6 +71,14 @@ protected FFbase()
                 return;
             }
 
+            string workingDirectory = Environment.CurrentDirectory;
```

Comment:
This won't work if you start app in other directory. 
[Look at this](https://stackoverflow.com/questions/52797/how-do-i-get-the-path-of-the-assembly-the-code-is-in)
It should looks like above to get ffmpeg executables from directory where running executable are
string workingDirectory = System.Reflection.Assembly.GetAssembly(typeof(this)).Location;